### ISSUE TITLE
Restoring Deeplab from checkpoints, what exacly heppenes with parameters?

### ISSUE THREAD

**Issue by urgentyolo** [tensorflow/models#8013]
Does Deeplab (https://github.com/tensorflow/models/tree/master/research/deeplab) restore learning rate from checkpoint or from command line argument, when running Deeplab restored from its checkpoint? (For example if I run Deeplab, then I kill the process and then run Deeplab again with same arguments)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by urgentyolo** [comment]
Yes of course it counts learning rate dependent of current global step.

**Comment by urgentyolo** [comment]
Unfortunately, doing training with breakes and restoring from checkpoints made my loss plot zigzag, with no breakes it was smooth (according to Tensorboard), don`t know why.
